### ISSUE TITLE
jsii.errors.JSIIError: Cannot redefine property: Symbol(__jsii__)

### ISSUE THREAD

**Issue by BenZoomin** [udondan/cdk-ec2-key-pair#19]
Hi, I'm using the Python version of the library.
Upon importing the construct using `from cdk_ec2_key_pair import KeyPair` I'm getting an error.
Here's the full output of `cdk deploy`:
```
jsii.errors.JavaScriptError: 
  TypeError: Cannot redefine property: Symbol(__jsii__)
      at Function.defineProperty (<anonymous>)
      at Object.tagJsiiConstructor (/Users/benfaingold/Projects/cdk-region/.env/lib/python3.8/site-packages/jsii/_embedded/jsii/jsii-runtime.js:6488:12)
      at Kernel._addAssembly (/Users/benfaingold/Projects/cdk-region/.env/lib/python3.8/site-packages/jsii/_embedded/jsii/jsii-runtime.js:7899:31)
      at Kernel.load (/Users/benfaingold/Projects/cdk-region/.env/lib/python3.8/site-packages/jsii/_embedded/jsii/jsii-runtime.js:7669:14)
      at KernelHost.processRequest (/Users/benfaingold/Projects/cdk-region/.env/lib/python3.8/site-packages/jsii/_embedded/jsii/jsii-runtime.js:7456:28)
      at KernelHost.run (/Users/benfaingold/Projects/cdk-region/.env/lib/python3.8/site-packages/jsii/_embedded/jsii/jsii-runtime.js:7394:14)
      at Immediate._onImmediate (/Users/benfaingold/Projects/cdk-region/.env/lib/python3.8/site-packages/jsii/_embedded/jsii/jsii-runtime.js:7397:37)
      at processImmediate (internal/timers.js:456:21)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "app.py", line 4, in <module>
    from cdk_region.cdk_region_stack import CdkRegionStack
  File "/Users/benfaingold/Projects/cdk-region/cdk_region/cdk_region_stack.py", line 5, in <module>
    from cdk_ec2_key_pair import KeyPair
  File "/Users/benfaingold/Projects/cdk-region/.env/lib/python3.8/site-packages/cdk_ec2_key_pair/__init__.py", line 70, in <module>
    from ._jsii import *
  File "/Users/benfaingold/Projects/cdk-region/.env/lib/python3.8/site-packages/cdk_ec2_key_pair/_jsii/__init__.py", line 16, in <module>
    import cdk_iam_floyd._jsii
  File "/Users/benfaingold/Projects/cdk-region/.env/lib/python3.8/site-packages/cdk_iam_floyd/__init__.py", line 494, in <module>
    from ._jsii import *
  File "/Users/benfaingold/Projects/cdk-region/.env/lib/python3.8/site-packages/cdk_iam_floyd/_jsii/__init__.py", line 13, in <module>
    __jsii_assembly__ = jsii.JSIIAssembly.load(
  File "/Users/benfaingold/Projects/cdk-region/.env/lib/python3.8/site-packages/jsii/_runtime.py", line 43, in load
    _kernel.load(assembly.name, assembly.version, os.fspath(assembly_path))
  File "/Users/benfaingold/Projects/cdk-region/.env/lib/python3.8/site-packages/jsii/_kernel/__init__.py", line 239, in load
    self.provider.load(LoadRequest(name=name, version=version, tarball=tarball))
  File "/Users/benfaingold/Projects/cdk-region/.env/lib/python3.8/site-packages/jsii/_kernel/providers/process.py", line 333, in load
    return self._process.send(request, LoadResponse)
  File "/Users/benfaingold/Projects/cdk-region/.env/lib/python3.8/site-packages/jsii/_kernel/providers/process.py", line 321, in send
    raise JSIIError(resp.error) from JavaScriptError(resp.stack)
jsii.errors.JSIIError: Cannot redefine property: Symbol(__jsii__)
Subprocess exited with error 1
```
Can you help me understand why this is happening?

**Comment by udondan** [comment]
Thanks for letting me know. That's well beyond my knowledge of JSII internals. I [created an issue on the JSII project](https://github.com/aws/jsii/issues/1983), hopefully they can point to the exact problem/solution.

**Comment by udondan** [comment]
CC @mbonig: I assume nightynight and wakeywakey will have the same problem.

**Comment by mbonig** [comment]
Very possible, I never touch-test my code in python though so I'm not sure.

**Comment by udondan** [comment]
@mbonig Exactly the situation here. I assumed "when it builds, it works". Apparently wrong assumption. 😼

**Comment by udondan** [comment]
I identified the cause for the problem. See the JSII issue above for details.

I just released version 1.7.1. Please let me know if you have experience further problems.
